### DISCUSSION TITLE
安装完插件后报asyncio.exceptions.CancelledError

### DISCUSSION THREAD

**Discussion by billyosoror** [nonebot/discussions#519] [问答] [answered]
在安装完某市场插件后，nb无法正常启动，会提示asyncio.exceptions.CancelledError错误，有尝试nb orm upgrade

![1](https://github.com/user-attachments/assets/0a5111ce-b17b-44ca-a1d4-90a31465194e)
![2](https://github.com/user-attachments/assets/3e6548b3-75d4-45fc-9bfd-a5fa31ac1bb6)
![3](https://github.com/user-attachments/assets/58c3346e-aa01-4461-9faf-01016e3ac509)

**Comment by yanyongyu** [comment, marked as answer]
你的插件需要数据库升级，请参考[orm文档](https://nonebot.dev/docs/best-practice/database/user)进行升级。`nb orm upgrade`
